wallet/pskt/src/convert.rs

```diff
@@ -53,3 +53,54 @@ impl TryFrom<TransactionOutput> for Output {
         Ok(output)
     }
 }
+
+impl From<(cctx::Transaction, Vec<(&cctx::TransactionInput, &cctx::UtxoEntry)>)> for Inner {
+    fn from((transaction, populated_inputs): (cctx::Transaction, Vec<(&cctx::TransactionInput, &cctx::UtxoEntry)>)) -> Inner {
+        let inputs = populated_inputs
+            .into_iter()
+            .map(|(input, utxo)| {
+                InputBuilder::default()
+                    .utxo_entry(utxo.to_owned().clone())
+                    .previous_outpoint(input.previous_outpoint)
+                    .sig_op_count(input.sig_op_count)
+                    .build()
+                    .unwrap()
+            })
+            .collect();
+
+        let outputs: Vec<Output> = transaction
+            .outputs
+            .iter()
+            .filter_map(|output| Output::try_from(TransactionOutput::from(output.to_owned())).ok())
```

Comment:
what about explicitly returning error if conversion is failed?

Reply:
Done!